encoding/field_value.go

```diff
@@ -81,9 +102,9 @@ func DecodeFieldValue(b []byte, descending bool) ([]byte, any, error) {
 			b, v, err = DecodeVarintAscending(b)
 		}
 		if err != nil {
-			return nil, nil, NewErrCanNotDecodeFieldValue(b, client.FieldKind_NILLABLE_INT, err)
```

Comment:
question: Why did you remove the Kind from the log, it is still known here and might be useful no?

Reply:
I removed it because at some point `cliend.FieldKind` was removed from the func signature.
I bought it back.